projects/templates/src/lib/services/po-page-customization/po-page-customization.service.ts

```diff
@@ -67,7 +68,9 @@ export class PoPageCustomizationService {
     if (prop.merge) {
       return this.mergeOptions(originalOption[prop.nameProp], newPageOptions[prop.nameProp], prop.keyForMerge);
     } else {
-      return newPageOptions[prop.nameProp] || originalOption[prop.nameProp];
+      return newPageOptions[prop.nameProp] !== undefined
```

Comment:
```suggestion
        return newPageOptions[prop.nameProp] ?? originalOption[prop.nameProp];
```

Dessa forma ele usará o originalOption quando o newPageOptions for undefined e nullo..

Isso quer dizer que se informar "false" ele aceitará.